README.md

```diff
@@ -80,7 +80,7 @@ We publish SBOMs with all of our releases as of 0.3.0. You can find them in Mave
 
 - support for various backend [providers](https://openfeature.dev/docs/reference/concepts/provider)
 - easy integration and extension via [hooks](https://openfeature.dev/docs/reference/concepts/hooks)
-- bool, string, numeric and object flag types
+- bool, string, numeric, and object flag types
```

Comment:
Do we need an oxford comma policy?

Reply:
I tend to prefer oxford commas, but I can revert this change.

Reply:
I was joking. I don't care at all.